guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -80,6 +115,25 @@ To add details to a compute profile, complete the following steps:
 . In the *Size (GB)* field, enter the size of the storage to create on the host.
 . Click *Submit* to save the compute profile.
 
+.For CLI Users
+
+. Create a compute profile to use with the GCE compute resource:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile create --name _gce_profile_
+----
+
+. Add GCE details to the compute profile. 
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer compute-profile values create --compute-profile _gce_profile_ \
```

Comment:
Looks like the '--image_id' option is missing ? @kavita, I see that is also missing from hammer values compute-profile create --help.

Reply:
To make consistent with other compute resources, user won't be able to configure "image" value at compute profile level. I have created ticket to remove this field from compute-resource form  -  https://projects.theforeman.org/issues/29183

Reply:
@kgaikwad, Is that an UI change ? And when are we planning to release that ?

Reply:
@jyejare, 
Yes, it is UI change/fix and it is merged now in upstream which will remove image field from compute-profile page so no need to pass --image_id while creating compute-profile.

Reply:
The `--image` option is not available with the `hammer compute-profile values create` command, but it is there in the `hammer host create` command. I am marking this conversation as resolved.